Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM rentpath/rp_node:8.9.1
+
+WORKDIR $HOME/react-ui
+COPY --chown=node:node package.json yarn.lock $HOME/react-ui/
+run chown -R node:node $HOME/react-ui
+
+USER root
+COPY --chown=node:node . $HOME/react-ui/
+RUN apt-get install -y nginx
+
+USER node
+RUN yarn config set registry https://registry.yarnpkg.com
+RUN yarn install
+
+ARG VERSION
+ARG COMMIT
+ARG BUILD_NUMBER
+
+RUN yarn run build-storybook
+
+USER root
+RUN cp -R storybook-static/* /var/www/html/
+
+EXPOSE 80
+
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
Could we switch back to the `node` user before running the container or do we need root?

Reply:
We need nginx to run as root. Just to be certain I just tried it locally and I get:
```